docs/discord.rst

```diff
@@ -47,6 +46,10 @@ Creating a Bot account is a pretty straightforward process.
 
         The possibilities are endless, so **do not share this token.**
 
+        If you accidentally leaked your token, **click "Regenerate" button as soon
```

Comment:
```suggestion
        If you accidentally leaked your token, click the "Regenerate" button as soon
```

This sentence doesn't really work well with emphasis (since it would stand out without context).